.github/workflows/test.yml

```diff
@@ -0,0 +1,31 @@
+name: Run Tests with Coverage
```

Comment:
Good job setting up GitHub actions! First submission I have seen to go this far.